src/main/java/org/sagebionetworks/bridge/dynamodb/DynamoSurveyDao.java

```diff
@@ -85,6 +91,8 @@ QueryBuilder setDeleted(boolean includeDeleted) {
             List<DynamoSurvey> dynamoSurveys = null;
             if (surveyGuid == null) {
                 dynamoSurveys = queryBySecondaryIndex();
+            } else if (surveyGuid.startsWith("identifier:")) { 
```

Comment:
We should move "identifier:" to a constant.

We should make this check case-insensitive.